README.md

```diff
@@ -26,6 +26,12 @@ Create an HPI file to install in Jenkins (HPI file will be in `target/slack.hpi`
 
     mvn package
 
+If you are having trouble getting mvn to package due to a test failure, please add this to your pom.xml
```

Comment:
Didn't intend for this to be part of this pull request as its intended for our teams who see my branch since this still hasn't been merged yet.